librosa/display.py

```diff
@@ -1733,18 +1780,18 @@ def __same_axes(x_axis, y_axis, xlim, ylim):
 
 
 def waveshow(
-    y,
+    y: np.ndarray,
     *,
-    sr=22050,
-    max_points=11025,
-    x_axis="time",
-    offset=0.0,
-    marker="",
-    where="post",
-    label=None,
-    ax=None,
-    **kwargs,
-):
+    sr: float = 22050,
+    max_points: int = 11025,
+    x_axis: Optional[str] = "time",
+    offset: float = 0.0,
+    marker: str = "",
```

Comment:
I think `marker` can be more than just str, according to https://matplotlib.org/stable/api/_as_gen/matplotlib.lines.Line2D.html#matplotlib.lines.Line2D.set_marker

Reply:
Done